spec/receipt_printer_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'receipt_printer'
+
+describe ReceiptPrinter do
+  subject(:receipt_printer) {described_class.new(restaurant_data, totals)}
+  let(:restaurant_data) {{:shopname => :the_coffee_connection,
+                          :address => :'123_lakeside_way',
+                          :phone => "16503600708"}}
```

Comment:
Style/BlockEndNewline: Expression at 7, 51 should be on its own line.<br>Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)